claat/util/util.go

```diff
@@ -30,3 +30,28 @@ func Unique(a []string) []string {
 	}
 	return res
 }
+
+// NormalizedSplit takes a string, removes spaces, splits it along a comma delimiter, then on each fragment, trims Unicode spaces
+// from both ends and converts them to lowercase. It returns a slice of the unique processed strings.
+func NormalizedSplit(s string) []string {
+	if s == "" {
+		return []string{}
+	}
+	s = stripSpaces(s)
+	strs := strings.Split(s, ",")
+	for k, v := range strs {
+		strs[k] = strings.ToLower(v)
+	}
+	return Unique(strs)
+}
+
+func stripSpaces(s string) string {
```

Comment:
Could we make use of http://go/godoc/strings/#TrimSpace instead?

Reply:
No, we cannot.  We need to remove all of the spaces, not just leading and trailing.